src/libaudgui/init.cc

```diff
@@ -344,6 +344,13 @@ EXPORT void audgui_init ()
     if (init_count ++)
         return;
 
+#if defined(USE_GTK3) && defined(G_OS_UNIX) && !defined(__APPLE__)
+    // Use X11/XWayland by default, but allow to overwrite it.
+    // Especially the Winamp interface is not usable yet on Wayland
+    // due to limitations regarding application-side window positioning.
+    g_setenv("GDK_BACKEND", "x11", false);
```

Comment:
Do we want a check here to see if `GDK_BACKEND` is already set?

Reply:
That's why I'm using `overwrite: false` as third parameter. If it is already set, the variable is not overwritten.
So if users want to use Wayland, they can do that with `GDK_BACKEND=wayland`.

Reply:
I completely missed the `false`. Makes sense.